test/ibmq/test_ibmq_job.py

```diff
@@ -221,6 +221,22 @@ def test_get_jobs_from_backend(self, provider):
         """Test retrieving jobs from a backend."""
         backend = least_busy(provider.backends())
 
+        start_time = time.time()
+        job_list = backend.jobs(limit=5, skip=0)
+        self.log.info('found %s jobs on backend %s',
+                      len(job_list), backend.name())
+
+        for job in job_list:
+            self.log.info('status: %s', job.status())
+            self.assertTrue(isinstance(job.job_id(), str))
+        self.log.info('time to get job statuses: %0.3f s',
```

Comment:
Can you take the chance to drop the `self.log.info` (and the `start_time`, etc) from this test and from `test_get_jobs_from_backend_service`? They are a bit of a relic from older times and they don't provide too much value - we will do a more comprehensive round later.